package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "ga-wdi-boston.node-template",
```

Comment:
Opened issue to change name to `ga-wdi-boston.cs-data-structures`
https://github.com/ga-wdi-boston/cs-data-structures/issues/8